rtnetlink/src/traffic_control/get.rs

```diff
@@ -90,3 +87,42 @@ impl TrafficClassGetRequest {
         }
     }
 }
+
+pub struct TrafficFilterGetRequest {
```

Comment:
I'm not too familiar with `tc`, so would you mind adding a docstring with the equivalent `tc` commands this request corresponds to?

Reply:
The document is at `Handle::traffic_filter()`:

```
    /// Create a new handle, specifically for traffic control filter requests
    /// (equivalent to `tc filter show dev <interface_name>` commands)
```